libcontainer/configs/validate/validator.go

```diff
@@ -29,6 +30,7 @@ type check func(config *configs.Config) error
 
 func (v *ConfigValidator) Validate(config *configs.Config) error {
 	checks := []check{
```

Comment:
`v.mounts` has to be removed from this slice

Reply:
arf... 🤦 good catch

Reply:
Fixed